services/workers/src/test/java/org/sagebionetworks/file/worker/FileHandleStreamWorkerIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package org.sagebionetworks.file.worker;
+
+import java.util.Iterator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.sagebionetworks.ids.IdGenerator;
+import org.sagebionetworks.ids.IdType;
+import org.sagebionetworks.repo.manager.UserManager;
+import org.sagebionetworks.repo.model.AuthorizationConstants.BOOTSTRAP_PRINCIPAL;
+import org.sagebionetworks.repo.model.UserInfo;
+import org.sagebionetworks.repo.model.athena.AthenaQueryResult;
+import org.sagebionetworks.repo.model.athena.AthenaSupport;
+import org.sagebionetworks.repo.model.dao.FileHandleDao;
+import org.sagebionetworks.repo.model.dbo.dao.TestUtils;
+import org.sagebionetworks.repo.model.file.FileHandle;
+import org.sagebionetworks.util.Pair;
+import org.sagebionetworks.util.TimeUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.amazonaws.services.glue.model.Database;
+import com.amazonaws.services.glue.model.Table;
+
+// Note that we disabled this test since delivery from kinesis is setup after 15 minutes. This was used to test the wiring of the workers.
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(locations = { "classpath:test-context.xml" })
+@Disabled
```

Comment:
can we use junit.assume(StackConfig.runExtendTests()) to disable based on the context or something like that.